README.md

```diff
@@ -178,6 +178,7 @@ public class MainApplication extends Application implements ReactApplication {
 ```javascript
 // require the module
 var RNFS = require('react-native-fs');
+//Requiring the module with ES6 syntax (import RNFS from 'react-native-fs') throws 'Cannot read property 'readDir' of undefined'
```

Comment:
i'd suggest to tell both methods to requiring/import the module. One for ES5 and one for ES6. Since most projects are based on ES6 in the meanwhile we should not show the ES5 syntax anymore so popular. What do you think?

Reply:
Well in my case actually I am using ES6 in my project, and use of 'import' threw error, so had to change it to 'require'. Does that makes sense?

Btw, I noticed after your reply, I see your name (as it is catchy :)) in RN fb group a lot, and recently in ethereum developers group as well. Will add over FB if it is ok?

Reply:
Funny :) If you want to continue this conversation somewhere else (like FB) and miss my answer, please tag me or leave the link here that i can find it.